docker-compose.yaml

```
@@ -19,3 +19,13 @@ services:
     restart: unless-stopped
     ports:
       - '6379:6379'
+  web-api:
```

서버에 redis, mysql dependency를 추가해주세요.
```suggestion
  web-api:
    depends_on:
      - mysql
      - redis
```